libs/CovidDatasets.py

```diff
@@ -288,6 +319,11 @@ def parse_country(country):
                     logging.info('Received a 404 for date {}. Ending iteration.'.format(snapshot_date))
                     break
                 raise
+            except urllib.error.URLError as e:
+                if isinstance(e.reason, FileNotFoundError):
+                    logging.info('Received a 404 for date {}. Ending iteration.'.format(snapshot_date))
```

Comment:
Does this actually log an error message? 

Update: Whoaaa didn't realize that it just logged to the root logger.

No real need, but would be good to make a logger instance for this module.  I generally do:

```
_logger = logging.getLogger(__name__)
```
right after import statements at the top.